tailwind.config.cjs

```diff
@@ -5,6 +5,8 @@ const TemporalColors = {
   secondary: '#64748b',
   // Custom Colors
   primaryText: '#141414',
+  historyBg: '#DCE4FB',
```

Comment:
Do we want to have a token dedicated to something so small? /cc @GiantRobots

Reply:
It's technically the same as the Running status badge. We do use that pattern elsewhere..

Reply:
I can rename it or we can move it to just in a style sheet